datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -175,6 +179,14 @@ export class DatasetEntity implements Entity<Dataset> {
                         },
                     },
                 },
+                {
+                    name: 'Access Management',
+                    component: AccessManagement,
+                    display: {
+                        visible: (_, _1) => this.isAccessManagementEnabled(),
```

Comment:
just use the hook here: 

```
                        visible: (_, _1) => useIsShowAccessManagementEnabled(),
```

Reply:
recieving below error
"React Hook "useIsShowAccessManagementEnabled" is called in function "visible" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter react-hooks/rules-of-hooks"

to resolve that we used isAccessManagementEnabled

Reply:
done update

Reply:
got it. here's what we want to do then... pass a new optional parameter into the function called "appConfig" in line 249 on EntityProfile.tsx

Reply:
we can use a context object actually, then the method can just use that instead

Reply:
hi @jjoyce0510 we used a context object still had hooks issue. as a solution  we called "appConfig" in datasetEntity.tsx